src/services/DisposableCollection.ts

```diff
@@ -18,7 +18,7 @@ export class DisposableCollection implements IDisposable {
       this._dispose(disposable);
     } else {
       for (disposable of this.disposables) {
-        disposable.dispose();
+        disposable?.dispose();
```

Comment:
why is this needed?

Reply:
`this.disposables` array had `undefined` as values.
I could have investigate why, but this was the easiest fix